docs/roadmap/index.html

```diff
@@ -69,28 +63,29 @@ <h2 id="next-steps">Next Steps</h2>
   <li>propose a new charter for a W3C WebAssembly Working Group</li>
   <li>graduate the <a href="https://github.com/llvm-mirror/llvm/tree/master/test/CodeGen/WebAssembly">WebAssembly LLVM backend</a> from experimental to stable (and update Emscripten)</li>
   <li>prototype additional WebAssembly integration into browser developer tools</li>
-  <li>finalize a WebAssembly <a href="https://github.com/WebAssembly/design/issues/112">logo</a></li>
   <li>Start work on <a href="/docs/future-features/">post-MVP features</a></li>
-  <li>Create a publicly-runnable conformance test suite</li>
 </ul>
 
-<h2 id="planned-changes-before-release">Planned Changes Before Release</h2>
+<h2 id="changes-since-the-browser-preview">Changes Since the Browser Preview</h2>
 
 <ul>
-  <li>At the end of the Browser Preview period, the
-	<a href="/docs/binary-encoding/#high-level-structure">binary version</a>
-	will be reset to <code class="highlighter-rouge">0x1</code> (at which point the version will be frozen and all new features will be added in a backwards-compatible manner, and be detected through feature-testing).</li>
+  <li>The <a href="/docs/binary-encoding/#high-level-structure">binary version</a>
+  of WebAssembly has been frozen at <code class="highlighter-rouge">0x1</code> (from this point forward
+  all new features will be added in a backwards-compatible manner,
+  and be detected through feature-testing). To update your
+  WebAssembly modules, upgrade to the latest version of Emscripten 
+  and recompile.</li>
```

Comment:
Is Emscripten the only way to generate WebAssembly binaries that we want to mention?

Is there an easy way to diagnose "obsolete" binaries (e.g. the error messages, etc). As suggested in #1006 it would be good to know which versions of which browser support `0x1` as well, this would ease the transition.

Which version of Emscripten? I believe it's 1.37.3.